components/radiogroup/overview.md

```diff
@@ -53,58 +55,61 @@ Chosen gender: @( ChosenGender == 0 ? "no selection yet" : ChosenGender.ToString
 }
 ````
 
->caption The result from the code snippet above
-
-![Radio Group First Look](images/radiogroup-overview.gif)
-
->caption Component namespace and reference
-
-See the [Component Reference]({%slug radiogroup-databind%}#component-reference) section in the Data Binding article for details and examples.
+## Data Binding
 
+The Blazor RadioGroup supports data binding to primitive types and a model. [Read more about the Blazor RadioGroup data binding]({%slug radiogroup-databind%}).
 
-## Features
+## Layout
 
-The RadioGroup provides the following features:
+The RadioGroup component provides two ways to render the list of options - in a *vertical* or in a *horizontal* fashion. [Read more about the Blazor RadioGroup layouts]({%slug radiogroup-layout%}).
 
-* `Class` - the CSS class that will be rendered on the main wrapping element of the component.
+## Label Position
 
-* `Data` - allows you to provide the data source. Required.
+The RadioGroup component provides two ways to render the labels of the radio buttons - *before* or *after* the buttons themselves. [Read more about the Blazor RadioGroup label position]({%slug radiogroup-label-position%}).
 
-* `Enabled` - whether the component is enabled.
+## Appearance Settings
 
-* `Id` - renders as the id attribute on the main wrapping `ul` element.
+The Blazor RadioGroup has settings for customizing the button appearance. [Read more about the Blazor RadioGroup settings]({%slug radiogroup-appearance%}).
 
-* `LabelPosition` - whether the labels render after or before the radio button itself.
+## Events
 
-* `Layout` - whether the buttons are rendered vertically or horizontally.
+The Blazor RadioGroup fires value change and blur events that you can handle and further customize its behavior. [Read more about the Blazor RadioGroup events]({%slug radiogroup-events%}).
```

Comment:
Simpler (shorter) items first - makes it easier to read. We are usually not customizing the component behavior, but the application behavior.

```suggestion
The Blazor RadioGroup fires blur and value change events to respond to user actions. [Read more about the Blazor RadioGroup events]({%slug radiogroup-events%}).
```